machine-learning/setup/00-data-download.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+# Use the OpenPBTA bucket as the default.
+URL=${OPENPBTA_URL:-https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/data}
+RELEASE=${OPENPBTA_RELEASE:-release-v16-20200320}
+
+# All of the OpenPBTA data will be located in data/open-pbta/download folder
+# (from root of module directory) - we won't include the release in the folder structure
+data_dir=../data/open-pbta/download
+
+# The md5sum file provides our single point of truth for which files are in a release.
+curl --create-dirs $URL/$RELEASE/md5sum.txt -o ${data_dir}/md5sum.txt -z ${data_dir}/md5sum.txt
```

Comment:
probably don't need `-z` for this script. That is checking the time stamp, which I don't think we need. (Did we ever for the checksum file?)

Reply:
Ha no probably not. Reminds me that I should probably add a comment about where this was adapted from.